tests/validation/cattlevalidationtest/core/test_native_docker.py

```diff
@@ -43,85 +44,104 @@ def native_name(random_str):
     return 'native-' + random_str
 
 
-def test_native_unmanaged_network(socat_containers, client, native_name,
-                                  pull_images):
+def test_native_net_blank(socat_containers, client, native_name, pull_images):
     docker_client = get_docker_client(host(client))
-    d_container = docker_client.create_container(NATIVE_TEST_IMAGE,
-                                                 name=native_name)
-    docker_client.start(d_container)
-    inspect = docker_client.inspect_container(d_container)
+    inspect = docker_client.create_container(NATIVE_TEST_IMAGE,
```

Comment:
Interesting that you pointed that out because I struggled with the naming. I wanted to disambiguate between a rancher contiainer and docker container. Previously, I was using d_container, but that felt ugly. Maybe I just go with `docker_container` even though it is long.